lib/SILOptimizer/Analysis/MemoryBehavior.cpp

```diff
@@ -154,6 +154,9 @@ class MemoryBehaviorVisitor
   }
   REFCOUNTINC_MEMBEHAVIOR_INST(StrongRetainInst)
   REFCOUNTINC_MEMBEHAVIOR_INST(RetainValueInst)
+#define UNCHECKED_REF_STORAGE(Name, ...)                                       \
+  REFCOUNTINC_MEMBEHAVIOR_INST(Name##RetainValueInst)                          \
```

Comment:
I added this b/c these are manual retain, release instructions and should be treated as ref counting membehavior insts.